jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/ZipUtilTest.java

```diff
@@ -34,6 +39,7 @@
 public class ZipUtilTest {
 
   @Rule public final TemporaryFolder tempFolder = new TemporaryFolder();
+  public final FixedClock clock = new FixedClock();
```

Comment:
I don't think this is used? And not sure if `FixedCLock` from `com.google.api.client.testing.http` is a public API.

Reply:
Ah yes sorry, cleaned this up